test/server/gameSystems/DrawSystem.spec.js

```diff
@@ -0,0 +1,50 @@
+describe('Drawing cards', function() {
+    integration(function() {
+        describe('When a player draws cards', function() {
```

Comment:
I want to start moving tests towards using the new scheme that Matchstick added since it allows autocomplete. I'll be doing a big PR soon to refactor the existing tests as well. Please take a look at the Baze Malbus test to see the changes, but it boils down to:

1. Switch from spec.js to spec.ts
2. Change to `integration(function(contextRef)` at the top
3. Add `const { context } = contextRef;` in each test case
4. Use `contextRef` instead of `this` everywhere